packages/app-store/cal-ai/api/_getAdd.ts

```diff
@@ -33,6 +33,17 @@ export async function getHandler(req: NextApiRequest, res: NextApiResponse) {
     },
   });
 
+  await fetch(`${process.env.CALAI_URL}/api/onboard`, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${process.env.CALAI_API_KEY}`,
```

Comment:
These env vars are showing up undefined locally. 🫤

Reply:
Not sure what is required to get the env.appStore vars to show up - seems to be databased?

Reply:
Will mark as ready for review - since if the env doesn't work - it only affects the onboarding email, but hopefully we can get this resolved.